contracts/TokenLocking.sol

```diff
@@ -46,12 +46,12 @@ contract TokenLocking is TokenLockingStorage, DSMath { // ignore-swc-123
     _;
   }
 
-  // Prevent reputation miners from withdrawing stake during the mining process.
-  modifier hashNotSubmitted(address _token) {
+  // Prevent those involved in a mining cycle withdrawing stake during the mining process.
+  modifier notInvolvedInMiningCycle(address _token) {
```

Comment:
Not critical now but worth wondering if this stake management functionality could be subsumed by the more general stake management grammar proposed in #757 ? Would be nice if so.

Reply:
I mean, maybe, but I think that's definitely out of the purview of this PR given it's not really related to rewarding users and that functionality isn't merged yet. If this PR added significant functionality that was affected by that, I'd be much keener but it'll require changes to the existing functionality so should probably be its own PR.

Reply:
Fair fair